source/i18n/rules.ttl

```diff
@@ -0,0 +1,27 @@
+prefix : <https://id.kb.se/vocab/>
+base <https://id.kb.se/>
+
+</i18n/rule/x0/lessing> a :LanguageTransformRules ;
+  :label "Translitterering enligt Lessing"@sv ;
+  :code "lessing" ;
+  :inCollection </i18n/rule/x0> .
+
+</i18n/rule/x0/skr-1980> a :LanguageTransformRules ;
+  :label "Translitterering enligt SKR 1980"@sv ;
+  :code "skr-1980" ;
+  :inCollection </i18n/rule/x0> .
+
+</i18n/rule/m0/iso-1995> a :LanguageTransformRules ;
+  :label "Translitterering enligt ISO 1995"@sv ;
+  :code "iso-1995" ;
+  :inCollection </i18n/rule/m0> .
+
+</i18n/rule/x0/btj> a :LanguageTransformRules ;
+  :label "Translitterering enligt BTJ"@sv ;
+  :code "btj" ;
+  :inCollection </i18n/rule/x0> .
+
+</i18n/rule/m0/alaloc> a :LanguageTransformRules ;
+  :label "Translitterering enligt ALA-LOC"@sv ;
```

Comment:
"ALA-LOC" is both transliteration and transcription. "Romanisering enligt ALA-LOC" and more specific labels on `TransformedLanguageContent`?

Reply:
Agreed. Label adjusted in 29e6837a78aa001fcee60b48f28a3559c983870c

Reply:
Yes, it could we wise to extend the models for the `TransformedLanguageForm` (if you meant those?) to specify this.